relay/README.md

```diff
@@ -87,54 +68,72 @@ Scene 2:
-determines which metaphorical tubes to connect to each other.
+`/p2p-circuit/p2p-circuit/<first relay>/<first hop multiaddr>/<destination peer multiaddr>`
 
 A few examples:
 
+Using any relay available:
+
 - `/p2p-circuit/p2p/QmTwo`
-  - Dial QmTwo, through any available relay node.
-  - The relay node will use peer routing to find an address for QmTwo.
-- `/p2p/QmRelay/p2p-circuit/p2p/QmTwo`
-  - Dial QmTwo, through QmRelay.
-  - Use peer routing to find an address for QmRelay.
-  - The relay node will also use peer routing, to find an address for QmTwo.
+  - Dial QmTwo, through any available relay node (or find one node that can relay).
+  - The relay node will use peer routing to find an address for QmTwo if it doesn't have a direct connection.
 - `/p2p-circuit/ip4/../tcp/../p2p/QmTwo`
   - Dial QmTwo, through any available relay node,
     but force the relay node to use the encapsulated `/ip4` multiaddr for connecting to QmTwo.
   - We'll probably not support forced addresses for now, just because it's complicated.
-- `/ip4/../tcp/../p2p/QmRelay/p2p-circuit`
-  - Listen for connections relayed through QmRelay.
+
+Specify a relay:
+
+- `/p2p-circuit/p2p/QmRelay/p2p/QmTwo`
+  - Dial QmTwo, through QmRelay.
+  - Use peer routing to find an address for QmRelay.
+  - The relay node will also use peer routing, to find an address for QmTwo.
+- `/p2p-circuit/ip4/../tcp/../p2p/QmRelay/p2p/QmTwo`
+  - Dial QmTwo, through QmRelay.
   - Includes info for connecting to QmRelay.
-  - Also makes QmRelay available for relayed dialing, based on how listeners currently relate to dialers.
-- `/p2p/QmRelay/p2p-circuit`
-  - Same as previous example, but use peer routing to find an address for QmRelay.
+  - The relay node will use peer routing to find an address for QmTwo.
+
+Double relay:
+
 - `/p2p-circuit/p2p/QmTwo/p2p-circuit/p2p/QmThree`
   - Dial QmThree, through a relayed connection to QmTwo.
   - The relay nodes will use peer routing to find an address for QmTwo and QmThree.
   - We'll probably not support nested relayed connections for now, there are edge cases to think of.
-- `/ip4/../tcp/../p2p/QmRelay/p2p-circuit/p2p/QmTwo`
-  - Dial QmTwo, through QmRelay.
+
+--
+
+?? I don't understand the usage of the following:
+
+- `/ip4/../tcp/../p2p/QmRelay/p2p-circuit`
+  - Listen for connections relayed through QmRelay.
   - Includes info for connecting to QmRelay.
-  - The relay node will use peer routing to find an address for QmTwo.
+  - Also makes QmRelay available for relayed dialing, based on how listeners currently relate to dialers.
+- `/p2p/QmRelay/p2p-circuit`
+  - Same as previous example, but use peer routing to find an address for QmRelay.
+
+?? I believe we don't need this one:
 - `/p2p-circuit`
   - Use relay discovery to find a suitable relay node. (Neither specified nor implemented.)
   - Listen for relayed connections.
   - Dial through the discovered relay node for any `/p2p-circuit` multiaddr.
```

Comment:
To me, the act of 'enabling relayed connections' is a config value and not a multiaddr. Do you agree @lgierth ?